headerFooter/style/MYHstatistikdel.css

```diff
@@ -0,0 +1,101 @@
+body {
+  margin: 0;
+  font-family: "Oswald", sans-serif;
+  width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  display: grid;
+  grid-gap: 1rem;
+  grid-template-columns: repeat(auto-fit, minmax(20ch, 1fr));
+}
+
+@media screen and (max-width: 400px) {
+  .container {
+    display: grid;
+    grid-gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(20ch, 1fr));
+  }
+}
+
+.container div {
+  padding: 10px;
+  border-bottom: solid black;
+  border-bottom-width: 1px;
+}
+
+.header {
+  grid-column-start: 1;
+  grid-column-end: 4;
+  font-size: 30px;
+  font-weight: bold;
+}
+
+.content-large {
+  grid-row-start: 2;
+  grid-row-end: 3;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  border-right: solid black;
+  border-right-width: 1px;
+}
+
+.content-small {
+  grid-row-start: 2;
+  grid-row-end: 3;
+  grid-column-start: 3;
+  grid-column-end: 4;
+}
+
+.mainText {
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.links1 {

```

Comment:
links1 är ett ganska dåligt namn, jag förstår inte när jag skall använda de här klassen, gäller även namnen på klassen nedan.

Försök hitta namn som beskriver vad det är för något som css-klassen kommer att göra. t.ex. om man lånar lite förenklat:

text-center är ett bra namn, den beskriver att texten blir centrerad.

article-part5-for-glory-the-reckoning är ett dåligt namn för samma sak, eftersom jag inte kan se hur jag skall använda klassen eller ens vad den gör.

Reply:
Ok, tack för din feedback. Har nu ändrat namnen på samtliga klasser så att de är tydligare. De har nu namn som "contentLeft", "contentLeft-text", "contentRight". Klasserna för länkarna har nu också bättre namn; som exempelvis "contentRight-links" > "yhutbildningarLink".